website/photos/services.py

```diff
@@ -130,7 +130,7 @@ def extract_photo(request, archive_file, photo, album):
 
     # Generate unique filename
     num = album.photo_set.count()
-    filename, extension = os.path.splitext(photo_filename)
+    __, extension = os.path.splitext(photo_filename)
```

Comment:
Why are two underscores used instead of one?

Reply:
`_` shadows the import of `lazy_gettext`